include/circt/Support/SymUniquer.h

```diff
@@ -0,0 +1,76 @@
+//===- SymUniquer.h - Declare Symbol Uniquer --------------------*- C++ -*-===//
```

Comment:
nit: but I'd spell out SymbolUniquer in the filename if that's what the main class is named. I guess I would have said the same about SymCache, but missed that.